obspy/clients/fdsn/tests/test_mass_downloader.py

```diff
@@ -303,7 +303,8 @@ def test_channel_priority_filtering(self):
         c2 = Channel("10", "SHE", time_intervals)
         c3 = Channel("00", "BHZ", time_intervals)
         c4 = Channel("", "HHE", time_intervals)
-        channels = [c1, c2, c3, c4]
+        c5 = Channel("", "ELZ", time_intervals)
+        channels = [c1, c2, c3, c4, c5]
```

Comment:
Did you mean to add some additional checks below this? Just adding one more channel does not add a lot to this particular test.